Update polymorphism_1.cs

christos-P opened the pull request:
# Title

Minor changes to make code more readable.

## Summary

Minor changes to make code more readable.

## Details

I propose we change the initialization of shapes using a Collection Initializer, instead of creating a List first and then Add one item after another. Also I propose we change the variable name in foreach from s to shape, since this I think it more meaningful. Last but not least I think using var instead of the exact types System.Collections.Generic.List<Shape> and Shape makes code more readable.

christos-P committed:
Update polymorphism_1.cs

I propose we change the initialization of shapes using a Collection Initializer, instead of creating a List first and then Add one item after another. Also I propose we change the variable name in foreach from s to shape, since this I think it more meaningful. Last but not least I think using var instead of the exact types System.Collections.Generic.List<Shape> and Shape makes code more readable.

dnfclas commented:
This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

svick commented on docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/polymorphism_1.cs:
```diff
@@ -49,16 +49,18 @@ static void Main(string[] args)
-            shapes.Add(new Rectangle());
-            shapes.Add(new Triangle());
-            shapes.Add(new Circle());
+            var shapes = new System.Collections.Generic.List<Shape>
```
Would it make sense to also simplify `System.Collections.Generic.List<Shape>` to just `List<Shape>`?

christos-P replied:
I totally agree with you ! Furthermore, I would suggest we include the using statement,
using System.Collections.Generic, at the top of our source file, in order when someone copy and paste it in an environment, in order to test it not to forget it.

svick replied:
In that case, I think that `using System;` (required for `Console`) should be added as well.

rpetrusha reviewed (changes requested):
Thanks for modernizing this code example, as well as for your contribution to the dotnet/docs repo, @christos-P. Just two changes -- add a `using` statement, and remove the namespace names in the List\<Shape> instantiations.

christos-P committed:
Update polymorphism_1.cs

Added Using statement at the top of the source file and simplified the declaration of shapes List.

christos-P commented:
@rpetrusha You are very welcome. I just dided the corresponding changes and I have commited them.

rpetrusha reviewed (approved):
Thanks for making the additional changes, @christos-P. We'll merge your PR now. The updated code should appear on docs.microsoft.com within a day or two.

christos-P commented:
@rpetrusha You are welcome !
